certified-connectors/The Bot Platform Connector/apiProperties.json

```diff
@@ -0,0 +1,35 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "eed20fda-1cf4-4657-93a9-d266ff0f10d5",
```

Comment:
@tbpsami secret look alike values are not allowed. Please replace with dummy data. You can use somthing like "Please enter your client ID".

Reply:
Done